src/utils/chartJs/CustomTooltip.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  tooltip: {
+    background: 'rgba(0, 0, 0, .8)',
+    color: 'white',
+    borderRadius: '4px',
+    pointerEvents: 'none',
+  },
+  tooltipKey: {
+    display: 'inline-block',
+    width: '10px',
+    height: '10px',
+    marginRight: '10px',
+  },
+  table: {
+    borderCollapse: 'collapse',
+  },
+};
+const Header = ({ xLabel }) => (
+  <thead>
+    <tr>
+      <th>{xLabel}</th>
+    </tr>
+  </thead>
+);
+const TopBodyLines = props => (
+  <React.Fragment>
+    <tr>
+      <td>
+        <span style={props.labelColors} className={props.classes.tooltipKey} />
+        {props.seriesLabel}: {props.currData}
+      </td>
+    </tr>
+    <tr>
+      <td>
+        {props.currData} ({props.higherOrLower})
+      </td>
+    </tr>
+  </React.Fragment>
+);
+
+function CustomTooltip({ classes, tooltipModel, series }) {
+  if (tooltipModel.opacity === 0) {
+    // Chartjs removes some properties in `tooltipModel` when datapoint
+    // is not active. Those propertiles includes `title`, `dataPoints`, etc.
+    // We can't go forward if that's the case
+    return null;
+  }
+
+  const { xLabel, index, datasetIndex } = tooltipModel.dataPoints[0];
+  const dataset = series[datasetIndex].data;
+  const seriesLabel = series[datasetIndex].label;
+  const labelColors = tooltipModel.labelColors[0];
+  const currData = tooltipModel.dataPoints[0].yLabel;
+  const higherIsBetter = !series[datasetIndex].meta.lower_is_better;
+  const higherOrLower = higherIsBetter ? 'higher is better' : 'lower is better';
+  const paddingStyle = {
+    padding: `${tooltipModel.yPadding}px ${tooltipModel.xPadding}px`,
+  };
+  const cmpProps = {
+    classes,
+    labelColors,
+    seriesLabel,
+    currData,
+    higherOrLower,
+  };
+
+  // console.log('series', series);
+  // console.log('tooltipModel', tooltipModel);
+
+  if (index === 0) {
+    return (
+      <div className={classes.tooltip} style={paddingStyle}>
+        <table className={classes.table}>
```

Comment:
Remove `<table>`, `<td>`, `<tr>`, etc.  replace with a single `<div>` and add styling if required.  This will make my inline suggestion easier.